src/index.css

```diff
@@ -39,7 +39,7 @@ code {
 } */
 
 #root h1 {
-  font-family: 'AlegreyaSansSC', sans-serif;
+  font-family: 'PressStart2P-Regular', sans-serif;
```

Comment:
I'm wondering if, for this title, we keep Alegreya so it's consistent with other branding. OR should we be moving all the Immers Space branding to PressStart2P?


![image](https://user-images.githubusercontent.com/10034859/228361474-93d0c8a1-6050-4c08-8eef-85354fdc47aa.png)